projects/ui/src/lib/components/po-menu/po-menu.component.spec.ts

```diff
@@ -644,19 +644,21 @@ describe('PoMenuComponent:', () => {
     });
 
     it('should show `po-menu-footer` if `hasFooter` is `true`', () => {
-      spyOnProperty(component, 'hasFooter').and.returnValue(true);
+      component.collapsed = true;
+      component.menus = [{ label: '1', icon: 'po-icon-user', shortLabel: '123', action: () => {} }];
 
-      fixture.detectChanges();
+      const footer = fixture.debugElement.nativeElement.querySelector('.po-menu-footer');
 
-      expect(nativeElement.querySelector('.po-menu-footer')).toBeTruthy();
+      expect(footer).toBe(null);
     });
 
     it('shouldn`t show `po-menu-footer` if `hasFooter` is `false`', () => {
```

Comment:
Verificar o teste.

A descrição do it não esta condizente com o que ele realmente faz.

```
it('should not show `po-menu-footer` if `collapsed` is `true` and `menus` are invalid', () => {
      component.collapsed = true;
      component.menus = [{ label: '1', icon: 'po-icon-user', action: () => {} }];

      fixture.detectChanges();
      const footer = fixture.debugElement.nativeElement.querySelector('.po-menu-footer');

      expect(footer).toBe(null);
});
```